packages/mdc-select/index.js

```diff
@@ -69,29 +71,40 @@ export class MDCSelect extends MDCComponent {
   }
 
   initialize(menuFactory = (el) => new MDCSimpleMenu(el)) {
-    this.menuEl_ = this.root_.querySelector('.mdc-select__menu');
+    this.surface_ = this.root_.querySelector(strings.SURFACE_SELECTOR);
+    this.label_ = this.root_.querySelector(strings.LABEL_SELECTOR);
+    this.bottomLine_ = this.root_.querySelector(strings.BOTTOM_LINE_SELECTOR);
+    this.selectedText_ = this.root_.querySelector(strings.SELECTED_TEXT_SELECTOR);
+    this.menuEl_ = this.root_.querySelector(strings.MENU_SELECTOR);
     this.menu_ = menuFactory(this.menuEl_);
-    this.selectedText_ = this.root_.querySelector('.mdc-select__selected-text');
+    this.ripple = new MDCRipple(this.surface_);
   }
 
   getDefaultFoundation() {
     return new MDCSelectFoundation({
       addClass: (className) => this.root_.classList.add(className),
       removeClass: (className) => this.root_.classList.remove(className),
+      addClassToLabel: (className) => this.label_.classList.add(className),
+      removeClassFromLabel: (className) => this.label_.classList.remove(className),
+      addClassToBottomLine: (className) => this.bottomLine_.classList.add(className),
+      removeClassFromBottomLine: (className) => this.bottomLine_.classList.remove(className),
+      setBottomLineAttr: (attr, value) => this.bottomLine_.setAttribute(attr, value),
       setAttr: (attr, value) => this.root_.setAttribute(attr, value),
       rmAttr: (attr, value) => this.root_.removeAttribute(attr, value),
-      computeBoundingRect: () => this.root_.getBoundingClientRect(),
+      computeBoundingRect: () => this.surface_.getBoundingClientRect(),
+      registerPointerDownHandler: (evtType, handler) => this.root_.addEventListener(evtType, handler),
```

Comment:
`evtType` is inconsistent with `type` used for existing `[de]registerInteractionHandler` immediately below.  (Also see question I raised in README about why these need to exist.)